deployer/src/infra/mod.rs

```diff
@@ -416,6 +417,7 @@ impl_component! {
     IoEngine,      4,
     CsiNode,       5,
     CsiController, 5,
+    HANodeAgent, 6,
```

Comment:
nit: Please align to the previous lines.
Should the HANodeAgent be 5 and run at the same time as the csi node, would it make sense to run them at the same time?